src/components/UserSelector.tsx

```diff
@@ -1,25 +1,28 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
 import { User } from '../types/User';
+import { useDispatch, useSelector } from 'react-redux';
+import { setAuthor } from '../features/author';
+import { RootState } from '../app/store';
 
 type Props = {
   value: User | null;
   onChange: (user: User) => void;
 };
 
-export const UserSelector: React.FC<Props> = ({
-  // `value` and `onChange` are traditional names for the form field
-  // `selectedUser` represents what actually stored here
-  value: selectedUser,
-  onChange,
-}) => {
+export const UserSelector: React.FC<Props> = ({}) => {
```

Comment:
The `UserSelector` component is missing props destructuring. Ensure that the props passed to the component are destructured and used appropriately within the component.